### DISCUSSION TITLE
Solar Eclipse Powerwall-Dashboard output with overlay from next day

### DISCUSSION THREAD

**Discussion by hulkster** [jasonacox/Powerwall-Dashboard#468] [Show and tell]
While I'm a bit late to share this, is anyone surprised (LOL!) that when there is a Solar Eclipse, output from your Solar Array drops.

![2024_04_08_combined_solar_radiation2](https://github.com/jasonacox/Powerwall-Dashboard/assets/145915975/f0f765a2-91f4-40be-92d6-ca6d845338dd)

Powerwall-Dashboard is AWESOME for showing this ... and I also overlaid the output from the next day which was very similar in terms of weather ... except no Solar Eclipse (or late clouds). 

My neighbor sent me a spreadsheet of the Solar Radiation data from his weather station (surprised it is that noisy) ... so I was able to graph that up in Excel and then overlay on top of Solar Output. It was a bit of a PIA getting them to line up correctly (times/units/lines/etc.) but the super flexibility of Powerwall-Dashboard made this a lot easier ... thanks for a great product!

**Comment by jasonacox** [comment]
Thanks @hulkster !  I love this. We clearly saw the dip on our system in Santa Clarita, CA.
<img width="1179" alt="image" src="https://github.com/jasonacox/Powerwall-Dashboard/assets/836718/551f5110-8830-4e30-af2f-d2f84bad495d">

**Reply by hulkster** [reply]
I forgot one label on my image (just re-posted) that on 4/8/2024 at 1240 the solar output was was 1.95kW ... and the next day at the same time it was 5.37kW.

That's a 64% reduction ... matching exactly the 64% maximum solar obscuration. 
Not surprising ... but kinda cool they matched.

**Comment by BJReplay** [comment]
I saw last year's total eclipse in Australia after responding to a luddite complaining falsely on social media (probably LinkedIn) that solar power wasn't predictable due to clouds and it's even worse when there are things like eclipses.  This was in around 2020.

I pointed out that for clouds there was SolCast and for Eclipses there was physics, and the next one in Australia was...

And I looked it up and decided that since it was a fabulous part of our country, I had to be there.  It was a fantastic trip, saw some amazing national parks in remote northwest Australia, swam with whale sharks off Ningaloo reef, and saw a fabulous hybrid solar eclipse (annular most of the path, but total over us).

**Comment by jgleigh** [comment]
<img width="1326" alt="image" src="https://github.com/jasonacox/Powerwall-Dashboard/assets/654130/783a1182-9498-4e5a-a2b5-e67ce84ae450">
Tucson, AZ

**Comment by youzer-name** [comment]
Near Washington, DC, showing just solar production and clouds:

![Eclipse solar production](https://github.com/jasonacox/Powerwall-Dashboard/assets/57046429/0f0b65b3-b1c2-4656-822d-092d3014dd3a)

I expect the drop-off would have been even more pronounced near Cleveland, OH, where I took this:

![20240408180827-sm](https://github.com/jasonacox/Powerwall-Dashboard/assets/57046429/61a43192-3945-471e-bf7b-f95e218ba5fe)

**Reply by jasonacox** [reply]
Wow!  Nice
